scripts/RunnerJenkinsfile

```diff
@@ -1,22 +1,22 @@
 node {
     checkout scm
 
-    sh 'chmod a+x scripts/make_aws_features'
-    sh 'bash scripts/make_aws_features'
+    sh 'chmod a+x scripts/make_features'
+    sh 'bash scripts/make_features'
 
     step ([$class: 'CopyArtifact',
                   projectName: params.cc_android_job,
-                  filter: '**/app-commcare-debug.apk',
+                  filter: '**/app-commcare-release.apk',
```

Comment:
didn't we need the `debug` build to test certain things involving/requiring the developer tools? How was this addressed?

Reply:
Seems like the earlier blocker behind using debug build was [enabling global privileges](https://github.com/dimagi/commcare-ui-tests/pull/115) through calabsh. Though I was not able to find which test actually fails because of that limitation. For me all ui tests are passing with release build so not sure if our earlier rationale for using debug build was right.